changelogs/fragments/56569-Support-recursive-removal-win_iis_virtualdirectory.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+- win_iis_virtualdirectory - "Support recursive removal as per issue #49755"
```

Comment:
The issue will be commented out due to `#`, either quote the entire entry or use the full URL without `#`.